fastlane/lib/fastlane/swift_fastlane_api_generator.rb

```diff
@@ -321,10 +321,7 @@ def ignore_param?(function_name: nil, param_name: nil)
     end
 
     def process_action(action: nil)
-      unless action.available_options
-        return nil
-      end
-      options = action.available_options
+      options = action.available_options or []
```

Comment:
OR? lol.

Reply:
Yeah yeah! `or` is totally okay when its not in a conditional I learned 🙃 That is the only case when its supposed to be used... when its not in boolean logic 😉

Reply:
In this case, it might be clearer to just use `||` thoughts?
I think since it's so uncommon to see `or` it might be better to use the OG `||`